packages/landing/src/attribution.ts

```diff
@@ -168,11 +168,18 @@ export class MapAttribution implements maplibre.IControl {
     const filteredLayerIds = filtered.map((x) => x.collection.id).join('_');
     Config.map.emit('visibleLayers', filteredLayerIds);
 
+    const licensor = (function (): string | null {
```

Comment:
This may be better to move into attribution, how about `renderLincense()` or similar?